pages/listings.tsx

```diff
@@ -1,7 +1,70 @@
-export default function ListingsPage() {
+import AuthGuard from "@App/lib/auth/AuthGuard";
+import useAuthContext from "@App/lib/auth/AuthContext";
+import fetchcurrentuserListings from "@App/lib/listings/useFetchCurrentUserListing";
+import { Card, CardMedia, Stack, CardContent, Box, LinearProgress, Paper, Typography, Button } from "@mui/material";
+import { updateCurrentUser } from "firebase/auth";
+import type { NextPage } from "next";
+import {
+  IBaseListing,
+  IListingAttributes,
+  TListingCollectionReference,
+  TListingDocumentReference,
+} from "@App/lib/listings/types"
+
+const ListingsPage: NextPage = () => {
+ 
+  const {currentUser} = useAuthContext()
+  const { data: listings, isLoading } = fetchcurrentuserListings(  { userId: currentUser?.id }  );
```

Comment:
this hook name should be pascalCase -> useFetchCurrentUserListings

Reply:
done